examples/player/simid_player.html

```diff
@@ -147,22 +153,38 @@
       <tr>
         <td>Ad type</td>
         <td>
-          <input type="radio" id="linear_ad" name="ad_type" checked> <label for="linear_ad" class="ad_type_label">Linear
+          <input type="radio" id="linear_ad" name="ad_type" onclick="toggleInputs()"> <label for="linear_ad" class="ad_type_label">Linear
         Ad</label>
-        <input type="radio" id="nonlinear_ad" name="ad_type"> <label for="nonlinear_ad" class="ad_type_label">Non-Linear
+        <input type="radio" id="nonlinear_ad" name="ad_type" onclick="toggleInputs()"> <label for="nonlinear_ad" class="ad_type_label">Non-Linear
           Ad</label>
         </td>
       </tr>
-      <tr>
-        <td>Interactive Creative File</td>
-        <td><input type="text" id="creative_url" size = 30 
-           value="../creatives/simid_overlay.html">
+      <tr id="creative_dimensions" class="hidden">
+        <td>Creative Dimensions</td>
+        <td>
+          <label for="x_val">X: </label>
+          <input type="number" id="x_val" value="5" min="0" class="resize_inputs">
+
+          <label for="y_val">Y: </label>
+          <input type="number" id="y_val" value="5" min="0" class="resize_inputs">
+
+          <label for="width">Width: </label>
+          <input type="number" id="width" value="500" min="0" class="resize_inputs">
+
+          <label for="height">Height: </label>
+          <input type="number" id="height" value="120" min="0" class="resize_inputs">
         </td>
       </tr>
-      <tr>
+      <tr id="media_file" class="hidden">
         <td>Media File</td>
         <td><input type="text" id="video_url" size = 30 
-           value="https://storage.googleapis.com/gvabox/media/red30s.mp4">>
+           value="https://storage.googleapis.com/gvabox/media/red30s.mp4">
```

Comment:
I know I commented on this in the other PR but it didn't get changed. We don't want storage.googleapis.com stuff in the SIMID samples. Is it possible to just put this file right into your pr?